.github/workflows/release.yml

```diff
@@ -9,6 +9,9 @@ on:
     branches:
       - master
 
+env:
+  NIXPKGS_ALLOW_BROKEN: 1
+
```

Comment:
Can we drop this and the `--impure`s from below and fix anything that breaks in the code instead?